schemas/security/SystemLog.schema

```diff
@@ -2,5 +2,6 @@
   User: { category: 'SystemUser', required: true },
   Category: { domain: 'Id', required: true },
   Timestamp: { domain: 'Timestamp', requires: true },
-  Operation: { domain: 'UserAccess', requires: true }
+  Operation: { domain: 'UserAccess', requires: true },
```

Comment:
Shouldn't it be `required`? (here and above)

Reply:
We will log certain operations, so I think each log record should refer to one

Reply:
You are right, I misspelled that